website/src/_posts/2022-09-3.0.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Uppy 3.0: Future-proof, conveniently easy, stable as ever" 
+date: 2022-09-01
+author: 
+  - aduh95
+  - arturi
+  - mifi
+  - murderlon
+image: "https://uppy.io/images/blog/3.0/uppy-3-0.jpg"
```

Comment:
Any reason to use a JPEG image instead of a PNG one?

Reply:
Usually size, in this case PNG was larger, so why not.